### PR TITLE
canceling all transactions on all markets for a market maker before e…

### PR DESCRIPTION
**Pull request by godmodegalactus**
…ach run

### CODE REVIEW

**Comment by godmodegalactus** [comment]
@mschneider approve?

**Comment by mschneider** [comment]
will push fixes tomorrow

File: src/confirmation_strategies.rs [hunk 1]
```diff
@@ -277,7 +280,7 @@ pub fn confirmations_by_blocks(
                 if block_slots.is_empty() {
                     continue;
                 }
-                start_block = *block_slots.last().unwrap();
+                start_block = *block_slots.last().unwrap() + 1;
```
Comment (mschneider) [inline src/confirmation_strategies.rs:283]: why is this one extended by 1?

Comment (godmodegalactus) [reply src/confirmation_strategies.rs:283]: This is to solve the duplicate blocks issue. We have already got all blocks till here, next time we will fetch last_block + 1 till end.

**Pull request merged**